src/LiveComponent/src/LiveCollectionTrait.php

```diff
@@ -46,6 +46,70 @@ public function removeCollectionItem(PropertyAccessorInterface $propertyAccessor
         $propertyAccessor->setValue($this->formValues, $propertyPath, $data);
     }
 
+    #[LiveAction]
+    public function moveCollectionItemUp(PropertyAccessorInterface $propertyAccessor, #[LiveArg] string $name, #[LiveArg] int $index): void
+    {
+        if ($index <= 0) {
+            return;
+        }
+
+        $propertyPath = $this->fieldNameToPropertyPath($name, $this->formName);
+        $data = $propertyAccessor->getValue($this->formValues, $propertyPath);
```

Comment:
After some thinking... I wonder if this should be a dedicated trait... 

Because right now someone using the trait will automatically have these methods available, even if they just want the add/delete (and reciprocally)